### ISSUE TITLE
Unable to locate phpcs

### ISSUE THREAD

**Issue by jcloutz** [ikappas/vscode-phpcs#43]
I am receiving the following error with phpcs installed locally and globally.

MacOS Sierra 10.12.4
VSCode 1.11.2

```
phpcs: Unable to locate phpcs. Please add phpcs to your global path or use composer dependency manager to install it in your project locally.
```

Terminal output:
```
$ phpcs --version
PHP_CodeSniffer version 2.8.1 (stable) by Squiz (http://www.squiz.net)
```

EDIT:
The above error occurs on a project hosted on a mounted drive accessible through a VPN.

I tried it on a local project and receive the following error
```
The PHP CodeSniffer Linter server crashed 5 times in the last 3 minutes. The server will not be restarted.
```
Console output:
```
linting started on: file:///path/to/file.php
```

**Comment by seanvree** [comment]
Yeah, I'm having this same problem.  


Anyone?

**Comment by jEstevezRod** [comment]
I have the same error.

**Comment by ikappas** [comment]
I just pushed some major new features in develop branch that need testing and include:

ability to set phpcs.executablePath
ability to set phpcs.showWarnings
ability to set phpcs.showSources
ability to enable phpcs.trace.server with messages or verbose option for tracing errors.
ability to set phpcs.composerJsonPath to something other that workspaceRoot/composer.json
Your feedback is very welcome!

**Comment by ikappas** [comment]
This issue should be resolved in latest develop branch so please consider testing your use cases.

**Comment by caramelomartins** [comment]
@ikappas I'm using Ubuntu 16.04 and I can confirm that I was able to go around this issue with version `1.0.0-beta.4`. My use cases worked and I was able to see warnings from `phpcs`and fix files. I still had to set the full path to my `phpcs` global alias, was that what you intended?

**Comment by ikappas** [comment]
I just released vscode-phpcs 1.0.0 which supports the VSCode 1.20+ and includes many enhancements most notable of which are:

1. multi root support
2. lint as you type support
3. automatic configuration search support
4. ability to set phpcs.executablePath
5. ability to set phpcs.showWarnings
7. ability to set phpcs.showSources
8. ability to set phpcs.composerJsonPath to something other that workspaceRoot/composer.json
9. ability to enable phpcs.trace.server with messages or verbose option for tracing errors.

I would like to give my thanks to all the people that helped track down issues in the previous betas and hope the new features help you get things done easier.

On that note, I am closing this issue to properly track any new issues in the new release. If you find this issue persists or any other issue please feel free to open a new ticket for v.1.0.0.
